### ISSUE TITLE
issue loading map

### ISSUE THREAD

**Issue by heman-theman** [anonymouspenguin000/Geoguessr-clone-old#1]
Hello

I have added my maps api key but I can't get past the mini map not loading.

Thanks

**Comment by anonymouspenguin000** [comment]
Hi! 😁

Which exactly map are you talking about? The one that appears immediately when you press the Start button, or the minimap in the right bottom corner to mark a guess? 🤔

The first map must be blank. Its purpose is to generate a random location. It's a hack, because then I hadn't found a good algorithm to generate a random Street View, so my solution was to require user interaction on a pre-loaded fragment on the 2D-map, but if the user sees the fragment then that would be a huge hint, so I made this map blank.
But if you are talking about the minimap for guessing that is in the corner - IDK why it could've not been loaded ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯

Actually, recently I've rewritten this clone on React and fixed many problems. You can find the new version in my profile 🙃
Repo: https://github.com/anonymouspenguin000/geoguessr-clone
App: https://geoguessr-clone.github.io/

However, you can message me on Telegram **@VL_PNK** , I would be glad to solve the problem (or just talk) 🗨️👋

**Comment by heman-theman** [comment]
Yes, the mini 2d map that pops up after you press start.  I can't get passed it.  Is there anyway to bypass this "mini map" and go straight to the random streetview location like geoguessr?  I downloaded and cloned the react version, it works in dev mode but when I build it it doesn't work. :(

**Comment by anonymouspenguin000** [comment]
As I mentioned before, that first minimap is to generate a random location and you cannot bypass it ¯⁠\⁠_⁠(⁠ ͡⁠°⁠ ͜⁠ʖ⁠ ͡⁠°⁠)⁠_⁠/⁠¯
You should read the tutorial on the homepage, it's below the "Start" form... When you see the blank map, you need to drag the Pegman (yellow guy in the corner) and drop it in a random place on the blank map. If nothing happened, try again in another place. If nothing happened again, click the "Refresh" button below the map and try again. And so on - drag, drop, refresh - until it says you "Wait for 15 seconds". And only then, after that 15-seconds interval, you can start playing. Yes, that approach is completely uncomfortable, but as I said, it was my best solution then 😶
I hope this helped. And if you want to fix problems with the React version, please create issues in that repository. But if you just want to play it, I have an already-built game on GitHub Pages - https://geoguessr-clone.github.io/
